package.json

```diff
@@ -69,13 +71,14 @@
         "eleventy-plugin-nesting-toc": "^1.3.0",
         "eleventy-plugin-page-assets": "^0.3.0",
         "eleventy-plugin-reading-time": "^0.0.1",
-        "eslint": "^8.19.0",
-        "eslint-config-eslint": "^7.0.0",
-        "eslint-plugin-jsdoc": "^39.2.0",
-        "eslint-plugin-jsx-a11y": "^6.5.1",
+        "eslint": "^8.56.0",
+        "eslint-config-eslint": "^9.0.0",
+        "eslint-plugin-jsdoc": "^48.0.4",
```

Comment:
```suggestion
```

This can be removed as it isn't used.

Note: package-lock.json should probably also be updated after this is removed.